src/planner/add_hashagg.c

```diff
@@ -20,12 +20,24 @@
 #include "compat/compat-msvc-exit.h"
 
 #include "compat/compat.h"
-#include "plan_add_hashagg.h"
+#include "planner.h"
 #include "import/planner.h"
 #include "utils.h"
 #include "guc.h"
 #include "estimate.h"
 
+/* This optimization adds a HashAggregate plan to many group by queries.
+ * In plain postgres, many time-series queries will not use a the hash aggregate
```

Comment:
```suggestion
 * In plain postgres, many time-series queries will not use a hash aggregate
```

Reply:
oh damn saw it too late will fix it in the next PR